ui/window.go

```diff
@@ -23,6 +23,7 @@ type Window struct {
 	walletInfo         *wallet.MultiWalletInfo
 	walletSyncStatus   *wallet.SyncStatus
 	walletTransactions *wallet.Transactions
+	transactionsWallet *wallet.TransactionsWallet
```

Comment:
Why is there `transactionsWallet` when `walletTransactions` contains the same information